cmd/testrunner.go

```diff
@@ -100,18 +107,31 @@ func testTypeCommandActionFactory(testType testrunner.TestType) cobraext.Command
 			return errors.Wrap(err, "fetching Elasticsearch client instance failed")
 		}
 
+		var results []testrunner.TestResult
 		for _, folder := range testFolders {
-			if err := testrunner.Run(testType, testrunner.TestOptions{
+			r, err := testrunner.Run(testType, testrunner.TestOptions{
 				TestFolder:         folder,
 				PackageRootPath:    packageRootPath,
 				GenerateTestResult: generateTestResult,
 				ESClient:           esClient,
-			}); err != nil {
+			})
+
+			results = append(results, r...)
+
+			if err != nil {
 				return errors.Wrapf(err, "error running package %s tests", testType)
 			}
 		}
 
+		report, err := testrunner.Report(testrunner.TestReporter(reporter), results)
+		if err != nil {
+			return errors.Wrap(err, "error generating test report")
+		}
+		cmd.Println("--- Test results: START ---")
+		fmt.Println(report)
```

Comment:
Did you verify if the verbose mode doesn't scramble the test output?

Reply:
Yes, I just verified this. The test output is printed out in its own section for each test type, after all the debug logs for that test type. I added an example of this to the PR description, for the `apache` package with the Console reporter.